app/api/v1/models/office_models.py

```diff
@@ -111,3 +118,16 @@ def fetch_an_office(pid):
         """ Fetch a political office by ID"""
         global POLITICAL_OFFICES
         return [office for office in POLITICAL_OFFICES if office['id'] == pid]
+
+    def office_reg_validadion(self):
```

Comment:
@artorious, kindly check the function name. Should be office_reg_validation()

Reply:
@Mutuba, @jaystaks, @sportview, @Nduhiu17 & @BRIGHTON-ASUMANI Thank you for the review and input. I have made changes to reflect feedback.